File: 3426-minimum-number-of-chairs-in-a-waiting-room/minimum-number-of-chairs-in-a-waiting-room.cpp
class Solution {
public:
    int minimumChairs(string s) {
        int chair = 0;
        int ans = 0;
        for(int i=0; i<s.length(); i++){
            if (s[i] == 'E') chair+=1;
            else chair-=1;

            ans = max(ans, chair);
        }
        return ans;
    }
};

File: 3426-minimum-number-of-chairs-in-a-waiting-room/minimum-number-of-chairs-in-a-waiting-room.go
func minimumChairs(s string) int {
    ans, chair := 0, 0
    for i:=0; i<len(s); i++ {
        if (s[i] == 'E'){
            chair += 1
        } else {
            chair -= 1
        }

        ans = max(ans, chair)
    }
    return ans
}